hardhat/contracts/PlanetFactory.sol

```diff
@@ -1,20 +1,10 @@
 //SPDX-License-Identifier: Unlicense
 pragma solidity >=0.8.0 <0.9.0;
 
-// import "usingtellor/contracts/UsingTellor.sol";
+import "hardhat/console.sol";
 
-// contract PlanetFactory is UsingTellor {
-contract PlanetFactory {
-
-    // Define planet types
-    struct PlanetCharacs {
-        uint radius;
-        string yield_source;
-        string image_link;
-    }
-    PlanetCharacs[] public planetTypes;
-
-    // Define planet information
+library SharedStructs {
```

Comment:
Will anything else go in SharedStructs other than planets?

Reply:
i think yield also , as  when we submit our amount in the vault , only part of the input yield will be getting the necessary yield required .